src/networks.ts

```diff
@@ -1,19 +1,35 @@
+import fs from "fs";
+import path from "path";
+
 export type NetworkName = "testnet" | "testnet-sepolia" | "testnet-sepolia-staging" | "staging" | "localhost";
 
 export interface NetworkInfo {
   url: string;
   abi?: string;
 }
 
+let RPC_API_KEY: string | null = null;
+const keyPath = path.join(process.cwd(), ".earthfast", "rpc_api_key");
+
+if (fs.existsSync(keyPath)) {
+  RPC_API_KEY = fs.readFileSync(keyPath, "utf-8").trim();
+} else if (process.env.RPC_API_KEY) {
```

Comment:
if you do npm run build and pass in the RPC_API_KEY as an environment variable does it have the correct value?

Reply:
I have some doubts regarding this, can we chat about it on tomorrow meeting

Reply:
Yes for sure